packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -208,35 +158,10 @@ describe('TagGroup', function () {
     expect(tag).toHaveAttribute('aria-label', 'Tag 1');
   });
 
-  it('no infinite loop if all keys are disabled', function () {
-    let {getAllByRole} = render(
-      <Provider theme={theme}>
-        <input type="text" id="foo" autoFocus />
-        <TagGroup aria-label="tag group" disabledKeys={['1', '2', '3']}>
-          <Item key="1" aria-label="Tag 1">Tag 1</Item>
-          <Item key="2" aria-label="Tag 2">Tag 2</Item>
-          <Item key="3" aria-label="Tag 3">Tag 3</Item>
-        </TagGroup>
-        <input type="text" id="bar" />
-      </Provider>
-    );
-
-    let tags = getAllByRole('gridcell');
-    tags.forEach((tag) => {
-      expect(tag).toHaveAttribute('tabIndex', '-1');
-    });
-    let inputs = getAllByRole('textbox');
-    expect(document.activeElement).toBe(inputs[0]);
-    userEvent.tab();
-    expect(document.activeElement).toBe(inputs[1]);
-    userEvent.tab({shift: true});
-    expect(document.activeElement).toBe(inputs[0]);
-  });
-
-  it('not disabled if extraneous keys are provided', function () {
+  it('not disabled if keys are provided', function () {
```

Comment:
IMO we can remove this test since disableKeys won't be in the types

Reply:
I left it as a test to confirm disabled tests don't work anymore, but can remove if you don't think that's necessary